activities/TankOp.activity/images/go-right.svg

```diff
@@ -43,15 +43,15 @@
      inkscape:current-layer="svg2" /><g
      display="block"
      id="go-right"
-     style="fill:#333333">
+     style="fill:#ffffff">
```

Comment:
You're not supposed to change this. It should be adapted automatically when disable/enable

Reply:
Modified files replaced with original svg files. Working as expected now.